website/docs/components/page-header/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,236 @@
+
+## Icon Tile
+
+Displays a Helios [IconTile](/components/icon-tile) as a visual indicator for the content of the page, object, or product branding.
+
+![Page Header Icon Tile](/assets/components/page-header/page-header-icon-tile.png =700x*)
+
+!!! Info
+
+For consistency and to avoid competing visually with the title, only the `medium` size of the [IconTile](/components/icon-tile) is supported in the Page Header.
+
+!!!
+
+## Badges
+
+Displays Helios [Badges](/components/badge) to communicate high-priority metadata like the status of the page and metadata that is subject to change. We recommend using a maximum of three Badges within the Page Header and prioritizing the most contextually relevant metadata.
+
+![Page Header Badges](/assets/components/page-header/page-header-badges.png =700x*)
+
+!!! Do
+
+Use badges to express similar or related metadata pertaining to the page content.
+
+![Page Header Badges Metadata](/assets/components/page-header/page-header-badges-do-metadata.png =700x*)
+
+!!!
+
+!!! Dont
+
+Don't use more than three badges within the Page Header, instead explore ways to express additional metadata in the `customMetadata` area or move the content to the main page.
+
+![Page Header Badges Metadata](/assets/components/page-header/page-header-badges-dont-many-badges.png =700x*)
+
+!!!
+
+## Subtitle
+
+Displays a subtitle beneath the title to communicate metadata that does not change frequently or is not subject to changes within the application. Common examples of this are organization name, project name, unique IDs, resource names, etc.
+
+![Page Header Subtitle](/assets/components/page-header/page-header-subtitle.png =700x*)
+
+!!! Dont
+
+Don't use full sentences in the subtitle, use a [description](#description) instead.
+
+![Page Header Full Sentence Subtitle](/assets/components/page-header/page-header-subtitle-dont-full-sentence.png =700x*)
+
+!!!
+
+## Description
+
+Displays a description beneath the title and subtitle to communicate more detailed information about the page, link out to external documentation and resources, and capture more generic information about the page. Not all pages need a description, especially if the title is explicit enough.
+
+We recommend limiting the Page Header description to 1-2 sentences. If a longer description is required, consider linking to external documentation or moving more detailed content to an interstitial component like a [Flyout](/components/flyout).
+
+![Page Header Description](/assets/components/page-header/page-header-description.png =700x*)
+
+!!! Dont
+
+Don't include overly complex details, long-form content, or instructions in the description. This can add too much visual weight to the Page Header and can detract from the content on the main page.
```

Comment:
```suggestion
Don’t include overly complex details, long-form content, or instructions in the description. This can add too much visual weight to the Page Header and can detract from the content on the main page.
```